packages/ia-components/sandbox/theatres/audio-1/audio-1.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types';
 
-import { MetadataService, Item } from 'ia-js-client'
+// import { MetadataService, Item } from 'ia-js-client'
```

Comment:
are you resurrecting this component? what is your use?

Reply:
Actually, it was at the very initial stage to the get-to-know iaux repo. so it might be related to upstream changes maybe. I was not resurrecting.